### PR TITLE
Deprecate project

### PR DESCRIPTION
**Pull request by pracucci**
**What this PR does**:
I was waiting to deprecate it because we wanted to donate it Cortex, but they already forked it, so I think we can just announce the deprecation of this project and move people to Grafana Mimir's jsonnet.

**Which issue(s) this PR fixes**:
N/A

**Checklist**
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by colega — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Deprecated
+
+This project has been deprecated and moved to [Grafana Mimir jsonnet](https://github.com/grafana/mimir/tree/main/operations/mimir).
```
Comment (pstibrany) [inline README.md:3]: Shall we link to Cortex's fork too?

**Review by pstibrany — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pracucci) [reply README.md:3]: Good idea.

Comment (pracucci) [reply README.md]: Done, WDYT?

Comment (pstibrany) [reply README.md]: Looks good, thank you.

**Pull request merged**